server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -180,21 +177,13 @@ private ConfigProperties() {
         DB_PASSWORD,
     };
 
-    // Pinsetter
-    public static final String TASKS = "pinsetter.tasks";
-    public static final String DEFAULT_TASKS = "pinsetter.default_tasks";
-    public static final String ENABLE_PINSETTER = "candlepin.pinsetter.enable";
```

Comment:
This property is also referenced in ./server/conf/candlepin.conf.template (and that is supposed to be set from the pinsetter_enabled property in ./server/build.gradle). So those entries should be removed. (Do we need to add entry for any of the new properties? It seems that we don't have a global equivalent property to enable/disable the asyn job system as a whole, and only job-specific, and white/black-list props?)

Reply:
The "pinsetter enable" setting never actually disabled all of pinsetter, but rather only the scheduling components. I've added that back in via candlepin.async.scheduler.enabled and updated the template accordingly.